.github/ISSUE_TEMPLATE/depedency-update-template.md

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
missing the stuff that goes at the top, like
```
---
name: Notify Bug Template
about: Create a report to help us improve
title: 'BUG: '
labels: Bug, Notify
assignees: ''
---
```

modify and add to template, labels should include QA

Reply:
Updated.